src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -503,6 +486,12 @@ public async Task<IActionResult> SaveAnswers(PageViewModel vm, Guid applicationI
         [HttpGet]
         public async Task<IActionResult> TaskList(Guid applicationId)
         {
+            var canUpdate = await CanUpdateApplication(applicationId);
+            if (!canUpdate)
+            {
+                return RedirectToAction("Applications");
```

Comment:
Does this view need updating with RoATP specific content? I don't think it's been touched since EPAO

Reply:
Nope, it's the TaskList for RoATP

Reply:
Applications() is the method that is executed once you've logged into RoATP